feature/s3/manager/download.go

```diff
@@ -464,6 +471,13 @@ func (d *downloader) setTotalBytes(resp *s3.GetObjectOutput) {
 	}
 }
 
+func (d *downloader) getETag() string {
+	d.m.Lock()
```

Comment:
If we set this with `sync.Once`, we don't need to grab the lock to read the value